qtawesome/icon_browser.py

```diff
@@ -113,7 +121,13 @@ def _updateFilter(self):
         if searchTerm:
             reString += ".*%s.*$" % searchTerm
 
-        self._proxyModel.setFilterRegExp(reString)
+        # QSortFilterProxyModel.setFilterRegExp has been removed in Qt 6.
+        # Instead, QSortFilterProxyModel.setFilterRegularExpression is
+        # supported in Qt 5.12 or later.
+        try:
+            self._proxyModel.setFilterRegExp(reString)
+        except:
```

Comment:
```suggestion
        except AttributeError:
```
Since the exception thrown by calling `self._proxyModel.setFilterRegExp(reString)` should be an `AttributeError`